Gemfile.lock

```diff
@@ -163,25 +166,25 @@ GEM
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     method_source (0.9.2)
-    mime-types (3.1)
+    mime-types (3.2.2)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2016.0521)
+    mime-types-data (3.2018.0812)
     mini_mime (1.0.1)
     mini_portile2 (2.3.0)
     minitest (5.11.3)
     multi_json (1.11.3)
-    mustermann (1.0.2)
-    newrelic_rpm (3.17.1.326)
+    mustermann (1.0.3)
+    newrelic_rpm (5.7.0.350)
```

Comment:
This is a really big jump. I like the idea of updating all of the gems frequently but would it be worth only installing Faker and removing FFaker. By doing a general gem update in a seperate pr as a practice we could make it easier to address breaking changes from our dependancies. I can go either way. It looks like the only things that made version jumps here are non critical. 

Thoughts?

Reply:
Yeah I could undo the changes and put up a separate PR. That might be easier in case something causes an error